packages/picasso/src/Autocomplete/Autocomplete.tsx

```diff
@@ -298,7 +302,14 @@ export const Autocomplete = forwardRef<HTMLInputElement, Props>(
             disabled={disabled}
             defaultValue={undefined}
             value={value}
-            ref={ref}
+            ref={node => {
+              inputRef.current = node
+              if (typeof ref === 'function') {
+                ref(node)
+              } else if (ref?.current) {
```

Comment:
Is this the correct logic? If `ref.current` is `null` shouldn't we set it?

Maybe you meant something like this:
```suggestion
              } else if (ref) {
```